afids_utils/transforms.py

```diff
@@ -88,3 +88,47 @@ def voxel_to_world(
         z=world_pos[2],
         desc=afid_voxel.desc,
     )
+
+
+def coord_system_xfm(
+    afid_set: AfidSet, new_coord_system: str = "RAS"
+) -> AfidSet:
+    """Convert AFID set between LPS and RAS coordinates
+
+    Parameters
+    ----------
+    afid_set
+        Object containing valid AfidSet
+
+    new_coord_sys
+        Convert AFID set to defined coordinate system (default: 'RAS')
+
+    Returns
+    -------
+    AfidSet
+        Object containing AFIDs stored in defined coordinate system
+
+    Raises
+    ------
+    ValueError
+        If invalid coordinate system, or if already in defined
+        cordinate system
+    """
+    if new_coord_system not in ["RAS", "LPS"]:
+        raise ValueError(
+            "Unrecognized coordinate system - please select RAS or LPS"
+        )
+
+    if afid_set.coord_system == new_coord_system:
+        raise ValueError(f"Already saved in {new_coord_system}")
+
+    # Create copy and update coordinate system
```

Comment:
I think this would be a little more concise as something like:

```python
new_afids = [attrs.evolve(afid, x=-afid.x, y=-afid.y) for afid in afid_set.afids]
return attrs.evolve(afid_set, coord_system=new_coord_system, afids=new_afids)
```

Reply:
Ahh, I really should dig deeper into the attrs library.